shacl12-core/index.html

```diff
@@ -1587,267 +1585,230 @@ <h3>Property Shapes</h3>
 						</div>
 					</div>
 				</aside>
+			</section>
+		</section>
+
+		<section id="expressions">
+			<h2>SHACL Expressions</h2>
+			<p><em>The introduction of this section is informative.</em></p>
```

Comment:
It would be better to always use distinct (sub)sections, where the parent (e.g., `id="expressions"`) is normative (informative sections may not have normative subsections), and each subsection (e.g., `id="expressions-intro"`, `id="property-paths"`, etc.) is labeled normative or informative, as appropriate.

Reply:
Not sure if this would really be better. I think this is a matter of preference.

Reply:
In this case, drop the `<p><em>...</em></p>`. 

I don't see anything in the intro that would have an effect. The intro is a overview.

Could s/Informally, both/Both/ if that worries you.

(Not this PR:: It looks to me like most uses of "Informally, " in the doc are unnecessary. Descriptive text can go in normative sections. The use of definition boxes makes it clear which text is the most significant.)